src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigAlignmentsConfigPicker.java

```diff
@@ -853,11 +863,14 @@ private static void removeDueToShortReadSpan(final List<AlignmentInterval> selec
         }
     }
 
+    @VisibleForTesting
+    static final int TEMPORARY_SELF_POINTING_SV_INTERVAL_CONTIG = 0;
```

Comment:
I don't understand this. It doesn't seem like you ever refer to that except when putting things into the max overlap map below. Why create these invalid intervals? Also, I think that our contig numbering scheme for `SVInterval` starts at zero so these fake intervals will look like real intervals. I think I'd much rather keep them the old way in `Tuple2<Integer, Integer>`s than jam them into invalid interval objects.